astisub/main.go

```diff
@@ -111,4 +120,5 @@ func main() {
 	default:
 		log.Fatalf("Invalid subcommand %s", cmd)
 	}
+
```

Comment:
I'm being picky here, but can you not add this line feed? It messes up the git diff

Reply:
Done